src/ai/backend/agent/agent.py

```diff
@@ -203,6 +203,7 @@ class AbstractKernelCreationContext(aobject, Generic[KernelObjectType]):
     kernel_features: FrozenSet[str]
     image_ref: ImageRef
     internal_data: Mapping[str, Any]
+    additional_syscalls: List[str]
```

Comment:
To make things more obvious, let's update the variable name a little bit:
```suggestion
    additional_allowed_syscalls: List[str]
```